manticore/ethereum.py

```diff
@@ -352,73 +352,99 @@ def make_function_arguments(*args):
     @staticmethod
     def make_function_call(method_name, *args):
         function_id = ABI.make_function_id(method_name)
-        def check_bitsize(value, size):
-            if isinstance(value, BitVec):
-                return value.size==size
-            return (value & ~((1<<size)-1)) == 0
         assert len(function_id) == 4
         result = [tuple(function_id)]
         result.append(ABI.make_function_arguments(*args))
         return reduce(lambda x,y: x+y, result)
 
+    @staticmethod
+    def get_uint(data, nbytes, offset):
+        """
+        Read a `nbytes` bytes long big endian unsigned integer from `data` starting at `offset` 
+
+        :param data: sliceable buffer; symbolic buffer of Eth ABI encoded data
+        :param offset: byte offset
+        :param nbytes: number of bytes to read starting from least significant byte
+        :rtype: int or Expression
+        """
+        nbytes = arithmetic_simplify(nbytes)
+        offset = arithmetic_simplify(offset)
+        padding = 32 - nbytes # for 160
```

Comment:
Why is padding 32 and what does `for 160` mean? Might be good to describe it more so that it is clear for ppl who look over this code (I am not into the topic of ethereum, maybe this is something very obvious, but still...)